### PR TITLE
Feature/5464 recommender

### PR DESCRIPTION
**Pull request by sinejespersen**
#### Link to issue

https://platform.dandigbib.org/issues/5464

#### Description

Add recommender design

#### Screenshot of the result

image

#### Checklist

- [ ] My complies with [our coding guidelines](../docs/code_guidelines.md).
- [ ] My code passes our static analysis suite. If not then I have added a comment explaining why this change should be exempt from the code standards and process.
- [ ] My code passes our continuous integration process. If not then I have added a comment explaining why this change should be exempt from the code standards and process.

#### Additional comments or questions

If you have any further comments or questions for the reviewer them please add them here.

### CODE REVIEW

**Comment by spaceo** [comment]
The are changes here possibly affecting the campaign. Have you checked if it is your changes that are triggering the Chromatic alarm?

**Review by spaceo — CHANGES_REQUESTED** [review]
Looks good. I have one comment about organizing BEM in sass.

File: src/stories/Library/recommender/recommender.scss [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+  background-color: $c-global-tertiary-2;
+  color: $c-text-primary-white;
+  border-color: $c-text-secondary-gray;
+
+  @include breakpoint-m {
+    padding: $s-6xl;
+  }
+}
+
+.recommender--bright {
```
[the remark this answers is not in the document]
Comment (sinejespersen) [reply src/stories/Library/recommender/recommender.scss]: done

**Review by jeppekroghitk — APPROVED** [review]

**Review by spaceo — APPROVED** [review]
LGTM 👍

**Pull request merged**